lib/manager/components/TransformationsViewer.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import React, { Component } from 'react'
+import { Col, ListGroup, Panel, Row, ListGroupItem, Label } from 'react-bootstrap'
+
+import type { FeedVersion, TableTransformResult } from '../../types'
+
+type Props = {
+  version: FeedVersion,
+}
+
+export default class TransformationsViewer extends Component<Props> {
+  _getBadge (transformResult: TableTransformResult) {
+    switch (transformResult.transformType) {
+      // ESLint thinks that these are form labels
+      /* eslint jsx-a11y/label-has-for: "off" */
+      case 'TABLE_MODIFIED':
+        return <Label bsStyle='primary'>Table Modified</Label>
+      case 'TABLE_ADDED':
+        return <Label bsStyle='success'>Table Added</Label>
+      case 'TABLE_REPLACED':
+        return <Label bsStyle='warning'>Table Replaced</Label>
+      case 'TABLE_DELETED':
+        return <Label bsStyle='danger'>Table Deleted</Label>
+    }
+  }
+
+  render () {
+    const {
+      version
+    } = this.props
+
+    if (version.feedTransformResult && version.feedTransformResult.tableTransformResults) {
+      const {tableTransformResults} = version.feedTransformResult
+      const transformContent = tableTransformResults.map(res => {
+        const badge = this._getBadge(res)
+        return (
+          <ListGroupItem key={res.tableName} style={{maxWidth: '720px'}}>
+            <Row>
+              <Col sm={8}>
+                <h4 style={{marginTop: '5px'}}>{res.tableName}</h4>
+              </Col>
+              <Col sm={4} style={{textAlign: 'right'}}>
+                <h4>{badge}</h4>
```

Comment:
I think you should place the label/badge next to the table name as you wrote it initially (it looks out of place otherwise).

Reply:
I didn't love the way the spacing looked with it on the left, but I'll move it back over since I'm not really sure I like it this way better either.